cfgov/v1/models/images.py

```diff
@@ -84,10 +84,19 @@ def resize(self, size):
         operation.run(mock_image, image=None, env={})
         return mock_image.width, mock_image.height
 
+    @property
```

Comment:
I remember us putting this in during pairing; did you leave it here purposefully? I thought we'd decided this didn't actually address the need.